safe_transaction_service/tokens/tasks.py

```diff
@@ -81,18 +81,32 @@ def calculate_token_eth_price_task(
                         ).eth_value
                         * underlying_token.quantity
                     )
-        if eth_price:
-            eth_value_with_timestamp = EthValueWithTimestamp(eth_price, now)
-            redis.setex(redis_key, redis_expiration_time, str(eth_value_with_timestamp))
-            if not getattr(settings, "CELERY_ALWAYS_EAGER", False):
-                # Recalculate price before cache expires and prevents recursion checking Celery Eager property
-                calculate_token_eth_price_task.apply_async(
-                    (token_address, redis_key),
-                    {"force_recalculation": True},
-                    countdown=redis_expiration_time - 300,
-                )
-        else:
-            logger.warning("Cannot calculate eth price for token=%s", token_address)
+        logger.debug("Calculated eth-price=%f for token=%s", eth_price, token_address)
```

Comment:
Maybe I'm getting this wrong, but this message is logged even if `eth_price` wasn't calculated, right?

Reply:
yes, when can't get price the previous functions returns `0.0`

Reply:
Ok, so I assume this would print `Calculated eth-price=0.0 for token=whatever` it's ok to me, just verifying this should print the line in both scenarios 👍🏻